dspace-server-webapp/src/main/java/org/dspace/app/rest/exception/DSpaceApiExceptionControllerAdvice.java

```diff
@@ -84,6 +85,12 @@ protected void handleWrongRequestException(HttpServletRequest request, HttpServl
         sendErrorResponse(request, response, ex, "Request is invalid or incorrect", HttpServletResponse.SC_BAD_REQUEST);
     }
 
+    @ExceptionHandler({MaxUploadSizeExceededException.class})
+    protected void handleMaxFileSizeExceeded(HttpServletRequest request, HttpServletResponse response,
+                                               Exception ex) throws IOException {
+        sendErrorResponse(request, response, ex, "Maximum upload size exceeded", HttpServletResponse.SC_BAD_REQUEST);
```

Comment:
https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html#SC_REQUEST_ENTITY_TOO_LARGE ?

Reply:
Changed to: SC_REQUEST_ENTITY_TOO_LARGE, thank you